<ui version="4.0" >
 <class>configUtil</class>
 <widget class="QWidget" name="configUtil" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SSVEPInterface Config Utility</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Windows width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="wWidth" >
       <property name="maximum" >
        <number>9999</number>
       </property>
       <property name="value" >
        <number>1280</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Windows height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="wHeight" >
       <property name="maximum" >
        <number>9999</number>
       </property>
       <property name="value" >
        <number>800</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="wFullscreen" >
       <property name="layoutDirection" >
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text" >
        <string>Enable fullscreen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QTableWidget" name="squaresTable" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <property name="styleSheet" >
        <string>background-color: none;</string>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy" >
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="alternatingRowColors" >
        <bool>false</bool>
       </property>
       <property name="showGrid" >
        <bool>true</bool>
       </property>
       <property name="gridStyle" >
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="rowCount" >
        <number>0</number>
       </property>
       <column>
        <property name="text" >
         <string>Frequency</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Screen frequency</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>X</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Y</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="editColorButton" >
       <property name="text" >
        <string>Pick new color</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="fillSquaresChecker" >
       <property name="layoutDirection" >
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text" >
        <string>Fill squares</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currColorLabel" >
       <property name="text" >
        <string>Current color:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>Add a square</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton" >
       <property name="text" >
        <string>Save &amp;&amp; exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
